doc/cli-interaction.md

```diff
@@ -0,0 +1,80 @@
+# ARGO Compute Engine CLI
+
+## Executable Scripts for cli interaction with the compute engine
+In the folder `/usr/libexec/ar-compute/bin/` reside executable scripts that can be used for uploading metric data and sync data to the hadoop cluster (HDFS Filesystem).
+
+|Script | Description | Shortcut|
+|upload_metric.py |The specific script is used in order to upload daily metric data (relative to a tenant) to HDFS. | <a href="#metric">Description</a>|
+|upload_sync.py |The specific script is used in order to upload daily sync data (relative to a tenant and a job) to HDFS. | <a href="#sync">Description</a>|
+|mongo_clean_ar.py |The specific script is used if necessary to clean a/r data from the datastore regarding a specific day full path.  | <a href="#ar">Description</a>|
+|mongo_clean_status.py |The specific script is used if necessary to clean status detail data from the datastore regarding a specific day. | <a href="#status">Description</a>|
+
+<a id="metric"></a>
+
+### upload_metric.py
+The specific script is used in order to upload daily metric data (relative to a tenant) to HDFS.
+
+#### Full path
+
+	/usr/libexec/ar-compute/bin/upload_metric.py
+
+#### Parameters
+
+| Type | Description | Required|
```

Comment:
I have not edited the cli-interaction yet. Will do so after we finish with the other two documents